docs/4.x/sending-emails.md

```diff
@@ -0,0 +1,101 @@
+---
+category: Develop
+---
+# Sending Emails
+
+Sometimes in Matomo or your plugin you'll want to send an email notifying one or more users of something happening, or
+providing them some useful or relevant information. In the code, this is done using the `Mail` class.
+
+This guide provides a brief description on how best to do that.
+
+## The Mail class
+
+All mail sending functionality in Matomo goes through the `Mail` class, which in turn uses the PHPMailer library.
+
+Using the class is straightforward:
+
+```php
+$config = \Piwik\Config::getInstance();
+
+$mail = new \Piwik\Mail();
+$mail->addTo($targetEmail, 'Matomo User');
+$mail->setFrom($config->General['noreply_email_address'], $config->General['noreply_email_name']);
```

Comment:
not  sure would we call `setDefaultFromPiwik` here instead? (could maybe also add `setDefaultFromMatomo` method proxying to `setDefaultFromPiwik`)

Reply:
I just took this from the test-email command. Will change it.